### DISCUSSION TITLE
SVTRv2 recognition accuracy drops to 0 after certain epochs

### DISCUSSION THREAD

**Discussion by amandhapola** [PaddlePaddle/PaddleOCR#15859] [General]
Hi,
I have been trying to finetune SVTRv2 [link](https://paddlepaddle.github.io/PaddleOCR/v2.10.0/en/algorithm/text_recognition/algorithm_rec_svtrv2.html) on custom dataset . the model link is [link](https://paddleocr.bj.bcebos.com/openatom/openatom_rec_svtrv2_ch_train.tar) . The training starts fine and the accuracy increases till 89%  for starting  epochs and then suddenly drops to 0 and remains there . My config
 file is as follows : 
`Global:
  debug: false
  use_gpu: true
  epoch_num: 100
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: /app/PaddleOCR/train_exps/exp4_add_5_datasets_currency_supersubscripts2_svtrv2
  save_epoch_step: 1
  eval_batch_step:
  - 0
  - 4000
  cal_metric_during_train: true
  pretrained_model: /app/PaddleOCR/best_accuracy_rec_svtrv2_ch_train/best_accuracy
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: /app/PaddleOCR/train_data/original_format/test_data/dataset11_15_86661426_89878311_2.jpg
  character_dict_path: /app/PaddleOCR/train_dict/en_dict.txt
  max_text_length: 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: /app/PaddleOCR/train_exps/exp4_add_5_datasets_currency_supersubscripts2_svtrv2
  d2s_train_image_shape:
  - 3
  - 48
  - 320
Optimizer:
  name: AdamW
  beta1: 0.9
  beta2: 0.999
  epsilon: 1.0e-08
  weight_decay: 0.05
  no_weight_decay_name: norm
  one_dim_param_no_weight_decay: true
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 5
Architecture:
  model_type: rec
  algorithm: SVTR_HGNet
  Transform: null
  Backbone:
    name: SVTRv2
    use_pos_embed: false
    dims:
    - 128
    - 256
    - 384
    depths:
    - 6
    - 6
    - 6
    num_heads:
    - 4
    - 8
    - 12
    mixer:
    - - Conv
      - Conv
      - Conv
      - Conv
      - Conv
      - Conv
    - - Conv
      - Conv
      - Global
      - Global
      - Global
      - Global
    - - Global
      - Global
      - Global
      - Global
      - Global
      - Global
    local_k:
    - - 5
      - 5
    - - 5
      - 5
    - - -1
      - -1
    sub_k:
    - - 2
      - 1
    - - 2
      - 1
    - - -1
      - -1
    last_stage: false
    use_pool: true
  Head:
    name: MultiHead
    head_list:
    - CTCHead:
        Neck:
          name: svtr
          dims: 256
          depth: 2
          hidden_dims: 256
          kernel_size:
          - 1
          - 3
          use_guide: true
        Head:
          fc_decay: 1.0e-05
    - NRTRHead:
        nrtr_dim: 384
        max_text_length: 25
        num_decoder_layers: 2
Loss:
  name: MultiLoss
  loss_config_list:
  - CTCLoss: null
  - NRTRLoss: null
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
Train:
  dataset:
    name: MultiScaleDataSet
    ds_width: false
    data_dir: /app/PaddleOCR/train_data/original_format
    label_file_list: /app/PaddleOCR/train_data/original_format/new_train_labels_1.txt
    ext_op_transform_idx: 1
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecAug: null
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  sampler:
    name: MultiScaleSampler
    scales:
    - - 320
      - 32
    - - 320
      - 48
    - - 320
      - 64
    first_bs: 8
    fix_bs: false
    divided_factor:
    - 8
    - 16
    is_training: true
  loader:
    shuffle: true
    batch_size_per_card: 1536
    drop_last: true
    num_workers: 4
    use_shared_memory: true
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: /app/PaddleOCR/train_data/original_format
    label_file_list: /app/PaddleOCR/train_data/original_format/new_test_labels_1.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape:
        - 3
        - 48
        - 320
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 512
    num_workers: 6
profiler_options: null
`
I am training on NVIDIA RTX 4000 20gb single GPU . RAM : 64 GB . OS : CentOS Stream 9
![svtrv2_acc_plot_1](https://github.com/user-attachments/assets/a934ca32-2d79-41d5-a007-1243cce835cb)
